README.md

```diff
@@ -70,24 +76,43 @@ You can follow the documentation on [how to use `farcasterd` with Docker](./doc/
 
 Running `monero-wallet-rpc` is also possible with Docker, see [run `monero rpc wallet`](./doc/docker-stack.md#run-monero-rpc-wallet) for more details.
 
-### :bulb: Use public infrastructure
+### Configuration
+
+`farcasterd` can be configured through a toml file located by default at `~/.farcaster/farcasterd.toml`, if no file is found `farcasterd` is launched with some default values. You can see an example [here](./farcasterd.toml).
```

Comment:
platform specific?

Reply:
`~/.farcaster/farcasterd.toml`

Reply:
You right, I'll generalized the doc to talk about others location of the default config file.

Reply:
Done in b527610b4dd86ae60af08f61b25d2c964f57cf43, is that better?